src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -101,8 +105,35 @@ export default function applyAuthInterface(httpClient, authConfig) {
       // Attempt to refresh the JWT cookies.
       return httpClient.refreshAccessToken()
         // Successfully refreshed the JWT cookies, fire the callback function.
-        .then(() => {
-          callback(httpClient.getDecodedAccessToken());
+        .then((response) => {
+          const refreshedAccessToken = httpClient.getDecodedAccessToken();
+
+          /* istanbul ignore next */
+          if (refreshedAccessToken === null) {
+            // This is the success block for refreshing an access token.
+            // Sometimes the access token is null for an unknown reason.
+            // Log here to learn more.
+            if (httpClient.loggingService && httpClient.loggingService.logError) {
+              httpClient.loggingService.logError(
+                new Error('Access token is null after refresh.'),
```

Comment:
FYI: `logError` will take a message as well, but this is fine. :) This is also fine to leave as an error, because it is definitely an error.